### DISCUSSION TITLE
Calendar component

### DISCUSSION THREAD

**Discussion by CodeAn** [livewire/flux#917] [Feature Requests] [answered]
Common component used in app design is a calendar. Not specifically talking about input type (however I see it could be re used) but general calendar component like this: 
https://tailwindui.com/components/application-ui/data-display/calendars

Just take a look at the code for each of the examples, calendars are always a nightmare to implement. 

I have had some thoughts on usage.

<flux:calendar variant="month" size="full" date="2024-12-22">
<flux:calendar.header>
// User can throw in some dropdowns for example
</flux:calendar.header>
<flux:calendar.events>
<flux:calendar.events.item start="2024-12-22 18.00" end="2024-12-22 19.00" title="Meeting">
Some more content, potentially HTML.
</flux:calendar.events.item>
</flux:calendar.events>
</flux:calendar>

date in the flux:calendar indicates what the currently selected view is. Since the variant is month, it would show december 2024. If variant would be day, it would show the full day for 22 december 2024.

**Comment by caballeroe359** [comment]
I was looking for more calendar options like this one. Hopefully the team can add this basic feature on the upcoming updates.

**Comment by jeffchown** [comment]
@caballeroe359 Have a look at recent posts in Caleb's X account - he's hard at work at a calendar component that is looking fantastic!

https://x.com/calebporzio

**Reply by caballeroe359** [reply]
I just checked it out I can't wait!

**Comment by joshhanley** [comment, marked as answer]
Going to close this as it has been released in Flux v2.0.
